spec/web_helpers.rb

```diff
@@ -1,7 +1,8 @@
-def create_space
+def create_space(name, description="")
```

Comment:
Probably? I mean, you wouldn't want to book a BnB without any sort of description, would you?